src/test_helpers/integration.cljs

```diff
@@ -66,10 +66,18 @@
   []
   @(rf/subscribe [:messenger/started?]))
 
+(defn wallet-loaded
```

Comment:
maybe `wallet-loaded?`

Reply:
Hm, also not sure. @J-Son89 any thoughs?

Reply:
? sounds good 👍

Reply:
Done =)
We just need the approvals and I can merge